include/dslash.h

```diff
@@ -125,12 +160,22 @@ namespace quda
       if (arg.pack_threads && arg.kernel_type == INTERIOR_KERNEL) {
         // if doing the fused kernel we tune how many blocks to use for
         // communication
-        constexpr int max_blocks_per_dir = 4;
+        constexpr int max_blocks_per_dir = 6;
```

Comment:
How carefully have we verified that some corners of the tuning space don't cause lockups b/c not enough blocks are active? Is this even a relevant question based on the design?